Update the catalyst 101 doc (#165)
catalyst-cooperative/catalystcoop-handbook: docs/catalyst_101.md

| **Paperwork**        | - **Sign contractor agreement**                                                                                                                                                                                                                                                                                                                                                                                                                                     | - **Buy \$1000 membership share**<br>- **Sign membership agreement**                                                                                                                                                                                                                                                                                                                                                                                                                                 |

> | **Meetings**         | - [M/W/Th stand-ups](mwth-stand-up-meetings-15-20-min)<br>- [Monday action-items](weekly-check-in--action-items-10-30-min)<br>- [Quarterly advisory committee meetings](quarterly-advisory-committee-meeting-1-hours)<br>- [Internal team meetings](internal-team-meetings)<br>- [Work team sprint meetings](work-team-meetings)<br>- [3 and 6 month feedback](contractor-feedback-meeting)<br>- [Financial check-in](monthly-financial-check-ins) (after 3 months) | - All contractor meetings, plus:<br>- [Annual or Special Member Meeting](member-meetings)<br>- [Board Meeting](weekly-board-meetings) (until 10+ members)<br>- [Certain feedback meetings for contractors](contractor-feedback-meeting)<br>- [Certain Feedback meetings for members](annual-member-feedback)                                                                                                                                                                                         |

Author: This looks like a giant change, but the only think I added here is a row for the work team sprints: 

```
[Work team sprint meetings](work-team-meetings)<br>- 
```

All co-op activities are in the domain of a particular team.

#### Current Work Teams

- **Caracal** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/22))
- **Margay** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/24))
- **Inframundo** ([project board](https://github.com/orgs/catalyst-cooperative/projects/9/views/6))

(internal-teams)=

#### Internal Teams

- **Admin**: Run taxes, invoicing, payroll.
- **Internal Management**: Curate PM tools and processes, track internal capacity, HR, documentation of processes.
- **Community Development**: Build community around PUDL development and data use. Improve user experience, communicate our mission to the world, and interact with users and contributors.
- **Grant Development**: Identify grant opportunities that are relevant to PUDL and
  Catalyst.
- **Business Development**: Identify opportunities and develop new relationships that lead to revenue-generating, mission-aligned client projects. Maintain relationships with existing clients.

> - **Hiring & New Employee Management**: Ad hoc team for hiring and onboarding new

Reviewer: We're missing PUDL Pat here, should we include it?

Author: I would say no. We can add that later if it becomes more of a thing.

- After running through the action items, we open the floor to discuss anything else relevant to the whole team. If you want to talk about something specific, add it to something to the agenda in the PUDL Huddle document under the appropriately titled meeting header (Ex: Weekly Meeting YYYY-MM-DD). If there isn’t one yet, feel free to make one. Sometimes there are no agenda items in which case we’ll end the meeting early.

(work-team-meetings)=

#### Weekly Sprint Meetings - Team Specific

**Attendees**: Team members.
**Notes:** See notes documents for specific teams: [work teams](https://drive.google.com/drive/folders/1CHbOJU6W3r3DE2LEQqbcfdme5DK-qufz)

- Work teams usually meet twice a sprint (once a week). First to plan the sprint and second to check-in half way through. See
  [above](sprint-planning) for more detail on these meetings.
- These meetings will consult the GitHub boards specific to each team.

(internal-team-meetings)=

#### Internal Team Meetings - Team Specific (time and occurrence differs)

**Attendees**: Team members.
**Notes:** See notes documents for specific teams: [Internal teams](https://drive.google.com/open?id=1KuF1sJGQMWmODpD5LhnWVN61Ymbmg9xR)

> - Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis to discuss and execute issues related to their domain. See [above](internal-teams) for a breif description, or refer to the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit) for a list of current teams, their responsibilities, and how often they meet.

Reviewer: ```suggestion
- Catalyst organizes internal tasks into a series of teams that meet on a semi-regular basis to discuss and execute issues related to their domain. See [above](internal-teams) for a brief description, or refer to the [Teams Index](https://docs.google.com/document/u/0/d/1B9wzU3IuNlJbqG-4XpcpJnIpXhq8IRzf6m4lJnxXVvo/edit) for a list of current teams, their responsibilities, and how often they meet.
```

> - Teams are not mandatory, but contractors are encouraged and members are expected to participate in at least one.

Reviewer: Would we approve a member candidate who refused to be in any internal teams? Would we be happy with a member who refused to be in any internal teams? From my perspective it seems like we wouldn't... but maybe that's a bigger discussion.

Author: I think we would definitely let people opt out of teams if we got large enough. Even now, if someone just wanted to code I think that would be fine--we have been talking about trying to reduce team size anyways.

> (quarterly-advisory-committee-meeting-1-hours)=

Reviewer: ```suggestion
```

No more advisory meeting, right? We can remove this label?

#### Weekly Board Meetings (30 min)

**Attendees**: Board members only.
**Notes:** [Governance Notes](https://docs.google.com/document/u/0/d/1m5m1gbFFYRpaWkKTFIvTSZ14HJT_jfGNDjneCNUP_ZI/edit)

- Until we have more than 10 members, the Board of Directors is made up of all of the members. Board meetings are the venue where decisions about the strategic direction and regular operation of the cooperative are made. Any member of the board can go in and add agenda items.

> - Board meetings may be cancled if there is nothing on the agenda.

Reviewer: ```suggestion
- Board meetings may be canceled if there is nothing on the agenda.
```

(annual-member-feedback)=

#### Annual Member Feedback (two 1 hour blocks)

**Attendees:** Appointed members and member being reviewed (reviewee).
**Notes:** [Member Feedback](https://drive.google.com/open?id=1zsKiLe1hgV3vuPOxuQZ7JljkRpj-PSOq)

- Catalyst members participate in annual check-ins where they have the opportunity to review the co-op as well as be reviewed by other members.
- These check-ins are preceded by written feedback from the reviewee about Catalyst and the other members about the reviewee pursuant to a form distributed a week before the feedback session. The first hour block is dedicated to co-op feedback for the reviewee while the second hour block is dedicated to the reviewee’s feedback for the co-op. These meetings may be back to back or scheduled separately a few days or hours apart.

(monthly-member-check-ins)=

#### Monthly Member Check-ins (30 min)

**Attendees:** Member and check-in facilitator.
**Notes:** Notes are private and only shared with the member and meeting facilitator.

- Catalyst members participate in monthly check-in calls where they

>   touch base on their annual goals and general feelings. These calls are intended to provide members with an outlet for expressing excitement or concern about anything going on.

Reviewer: ```suggestion
  touch base on their annual goals and general feelings. These calls are intended to help members keep track of their goals and make sure they are supported in achieving them.
```

Though you are right in that a lot of the support turns out to be providing an outlet to talk about work feels.
